src/helm.rs

```diff
@@ -26,7 +26,7 @@ pub struct HelmChart {
     pub name: String,
     pub repo: HelmChartRepo,
     pub version: String,
-    pub values: serde_yaml::Value,
+    pub options: serde_yaml::Value,
```

Comment:
How come?

Reply:
This is needed because otherwise the deserialization fails. The templated stacks use `options`, e.g. [here](https://github.com/stackabletech/stackablectl/blob/main/stacks/_templates/jupyterhub.yaml#L7).

Reply:
Well that sucks, but I can see why we're kind of stuck with it now...

Reply:
Agreed. We would have to change that in the old `stackablectl` and the stack files. We can do that once we migrate the the new one and the improved demo/stack/release repo structure.